packages/core/src/ui/widgets/colorBar/ColorBarScale.ts

```diff
@@ -0,0 +1,398 @@
+import { ColorBarRange } from './types/ColorBarRange';
+import { ColorBarVOIRange } from './types/ColorBarVOIRange';
+import { ColorBarSize } from './types/ColorBarSize';
+import { ColorBarScaleProps } from './types/ColorBarScaleProps';
+import { ColorBarPosition } from './types/ColorBarPosition';
+import isSizeValid from './common/isSizeValid';
+import isRangeValid from './common/isRangeValid';
+import rangesEqual from './common/rangesEqual';
+import sizesEqual from './common/sizesEqual';
+import positionsEqual from './common/positionsEquals';
+import { ColorBarScalePosition } from './enums/ColorBarScalePosition';
+
+const DEFAULT_FONT = '10px Arial';
```

Comment:
this would be much cleaner

```js
const DEFAULTS = {
  FONT: '10px Arial',
  COLOR: 'white',
  TICK_SIZE: 5,
  TICK_WIDTH: 1,
  TICK_LABEL_MARGIN: 3,
  MAX_NUM_TICKS: 8,
};
```